components/atom/table/src/index.js

```diff
@@ -43,7 +58,9 @@ const AtomTable = ({head, body, foot, fullWidth}) => {
                 colspan = 1
               } = cell
               const cellClassName = cx(`${baseClass}-cell`, {
-                [`${baseClass}-cell--noWrap`]: isNowrap
+                [`${baseClass}-cell--noWrap`]: isNowrap,
+                [`${baseClass}-cell--${cellPadding}`]: cellPadding,
```

Comment:
if cellpadding is undefined this will be added. Better removing this css class value if there is no defined cellpadding

Reply:
Yep, the same in the classname above. In order to prevent it, if `cellPadding` prop is not required, it could be fine to set a default value, don't you think?

Reply:
Yes, just added...
```
const AtomTable = ({
  head,
  body,
  foot,
  fullWidth,
  cellPadding = false,
  borderBottom = false
}) => {
```

Reply:
```js
cx(`${baseClass}-cell`,
  {
    [`${baseClass}-cell--noWrap`]: isNowrap,	                		 
    ... cellPadding && {[`${baseClass}-cell--${cellPadding}`]: cellPadding},
    ... borderBottom && {[`${baseClass}-cell--borderBottom`]: borderBottom},
  })
})
```

Reply:
only add classes if props are defined

Reply:
If `cellPadding` is `false`, the classname won't be added, not needed extra check, could you test it @nucliweb?

Reply:
true, we where just talking about that.